lib/markus_configurator.rb

```diff
@@ -59,7 +59,7 @@ def markus_config_repository_external_base_url
     if defined? REPOSITORY_EXTERNAL_BASE_URL
       return REPOSITORY_EXTERNAL_BASE_URL
     else
-      return 'http://www.example.com/git'
+      return "http://www.example.com/git"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.